scripts/src/main/resources/scripts/command/ide

```diff
@@ -3,6 +3,18 @@
 source "$(dirname "${0}")"/../functions
 DEVON_IDE_REPO_URL="https://repo.maven.apache.org/maven2/com/devonfw/tools/ide"
 
+function doUpdateMetaFile() {
+  if [ -f "${USERPROFILE}/devon.txt" ]
+  then
+    if ! grep -q "${DEVON_IDE_HOME}" "${USERPROFILE}/devon.txt"
+	then
+      echo "${DEVON_IDE_HOME};" >> "${USERPROFILE}/devon.txt"
+	fi
+  else
+    echo "${DEVON_IDE_HOME};" > "${USERPROFILE}/devon.txt"
+  fi
+}
```

Comment:
Excellent solution. However, suggestion for improvement:
* Please follow our guidelines and link commits and PRs with the according issues: https://github.com/devonfw/.github/blob/master/CONTRIBUTING.asciidoc#committing-changes - The issue this PR is implementing seems to be #416
* Also you should always create a fork to work on. On that create feature branches and from that create your PR. I have no clue why you have direct permission to commit on this repo, but I will try to restrict those for all users to prevent accidents in the future. See https://github.com/devonfw/.github/blob/master/CONTRIBUTING.asciidoc#forking
* We already have a `.devon` folder that we create in the users home directory. Hence we should write the medata data into that directory as already described in #416
* The filename `devon.txt` says nothing about its content. Instead it should be called `installations`, `ide-paths`, or something like this.
* Why do you terminate each line with a semicolon? I think this is a bad idea. It has to be removed when parsed and in odd cases a path may for whatever reason contain a semicolon.
* To improve, I would suggest using a local variable for the metafile to avoid repeating it four times redundantly.